lib/record-events-browser.js

```diff
@@ -132,6 +132,16 @@ export function recordEvents(eventsHash, callback){
   return send.call(this, { url, extendedEventsHash, callback });
 }
 
+// ------------------------------
+// .setOptOut
+// ------------------------------
+
+export function setOptOut(optOut = true) {
+  if (optOut) {
+    localStorage.setItem('optout', optOut);
+  }
```

Comment:
we need to handle 2nd possibility - if optOut param is false.
Scenario:
1. let's imagine that someone is reading new york times. he has chosen to optOut from tracking when he saw a popup.
2. few months later new york times convinces him to register, and he /by accepting the terms&conditions/ allowed us to track him. so we need to remove the localstorage value using client.setOptOut(false)